lib/tasks/bulk_import_documents_from_csv.rake

```
@@ -1,3 +1,5 @@
+desc "Import design decisions from a CSV file and save them with optional attachments"
```

Querying the commit message here: `Skip rows in the csv where required fields are missing`

Can you elaborate? Wouldn't we want to raise an exception if a required field is missing?

Is it worth doing a full validation of all the CSV contents before attempting the migration, so that we don't have to deal with semi-successful imports?

---

As discussed I have added the ability to do a dry run, we don't really know all the issues with the data until we try to import. If dry run is not selected the task will raise an error and not save any Design Decisions